scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly PROGDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly DIR="$(cd "${PROGDIR}/.." && pwd)"
+
+GOOS="linux" go build -ldflags='-s -w' -o bin/helper github.com/paketo-buildpacks/libjvm/v2/cmd/helper
+GOOS="linux" go build -ldflags='-s -w' -o bin/main "${DIR}/cmd"
```

Comment:
We just hard code it in other buildpacks: https://github.com/paketo-buildpacks/bellsoft-liberica/blob/main/scripts/build.sh#L5-L6

Reply:
iirc, I borrowed the build.sh from a non octo project.. I had an issue with the one in the other java projects.. but I don't recall what that was.. I can try putting one back in locally & see where the issue was =)

Reply:
OK, let me know what you find out. If there's a difference it would be good to document.

Reply:
Went digging.. User Error ;) The script without PROGDIR/DIR was failing as it couldn't find the package in github to build.. I figured this was because the code wasn't in github yet, but was actually because my main.go was at cmd/main.go rather than cmd/main/main.go =) Fixed, and updated in latest commit removing PROGDIR/DIR.